docker-compose.yml

```diff
@@ -173,7 +175,7 @@ services:
       GF_SERVER_ROOT_URL: 'https://%(domain)s/grafana/'
       GF_SMTP_ENABLED: ${TTN_DASHBOARD_GRAFANA_SMTP_ENABLED:-false}
       GF_SMTP_SKIP_VERIFY: ${TTN_DASHBOARD_GRAFANA_SMTP_SKIP_VERIFY:-false}
-      GF_SMTP_HOST: ${TTN_DASHBOARD_GRAFANA_SMTP_HOST:-localhost}:25
+      GF_SMTP_HOST: ${TTN_DASHBOARD_GRAFANA_SMTP_HOST:-localhost.com}:25
```

Comment:
I don't think this should be localhost.com... localhost is the full name in this case.